### PR TITLE
file instructions in readme

### LINKED ISSUE #4: Add wiki page on how to submit new data and what data are acceptable
Following the discussion in pr-omethe-us/PyKED#55, I propose that we add a wiki page to this repository with instructions for how to generate ChemKED files and submit them to the database. In addition, we should include criteria for data to be included.

### PR DESCRIPTION
**Pull request by mefuller**
closes #4

### CODE REVIEW

**Comment by kyleniemeyer** [comment]
looks good to me! @bryanwweber?

File: README.md [hunk 1]
```diff
@@ -5,3 +5,17 @@ License
 -------
 <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
 See the LICENSE.txt file or follow the link for details.
+
+Submissions: Sine qua none
+--------------------------
+1. All yaml files require a permanent link, prefereably a DOI, although arXiv identifiers should also be sufficient. These identifiers must be present in each and every file.
+2. Each file requires at least one identified "file-authors", preferable with ORCID (or perhaps some other permanent identifier) to establish a corresponding author for the dataset. This is to ensure that missing or malformed information may be corrected in the future.
```
Comment (bryanwweber) [inline README.md]:
```suggestion
2. Each file requires at least one identified person in the `"file-authors"` field, preferable with ORCID to establish a corresponding author for the dataset. This is to ensure that missing or malformed information may be corrected in the future.
```

File: README.md [hunk 2]
```diff
@@ -5,3 +5,17 @@ License
...
+3.  Blank fields are not permitted. In the example of a preprint/just accepted article, with no page or volume numbers available, do not include these fields. They may be appended at a later date.
+4.  Sanitize strings, especially SMILES identifiers SMILES: '[O][O]' or SMILES: 'N#N' clearly identifies the code as a string, while SMILES: [O][O] or SMILES: N#N do not and may (read: all but assuredly) lead to errors.
```
Comment (bryanwweber) [inline README.md]:
```suggestion
4.  Sanitize strings, especially SMILES identifiers, by enclosing them in quotes. `SMILES: '[O][O]'` or `SMILES: 'N#N'` clearly identifies the code as a string, while `SMILES: [O][O]` or `SMILES: N#N` (note the missing quote marks, `'`) will not be identified as strings and will lead to errors.
```

**Review by bryanwweber — CHANGES_REQUESTED** [review]
Some minor copyediting

**Comment by mefuller** [comment]
I'm fine with those changes and accepted them.
Can you squash on merge or do I need to rebase and squash?

**Review by bryanwweber — APPROVED** [review]

**Comment by bryanwweber** [comment]
No need to squash, thanks @mefuller !

**Pull request merged**